v1/src/test/java/com/google/cloud/teleport/spanner/ddl/DdlTest.java

```diff
@@ -252,6 +252,67 @@ public void pgSimple() {
     assertNotNull(ddl.hashCode());
   }
 
+  @Test
+  public void embeddingVector() {
+    Ddl.Builder builder = Ddl.builder();
+    builder
+        .createTable("Users")
+        .column("id")
+        .int64()
+        .notNull()
+        .endColumn()
+        .column("embedding_vector")
+        .type(Type.array(Type.float64()))
+        .arrayLength(Integer.valueOf(128))
+        .endColumn()
+        .primaryKey()
+        .asc("id")
+        .end()
+        .endTable();
+
+    Ddl ddl = builder.build();
+    assertThat(
+        ddl.prettyPrint(),
+        equalToCompressingWhiteSpace("CREATE TABLE `Users` ("
+                + " `id` INT64 NOT NULL,"
+                + " `embedding_vector` ARRAY<FLOAT64>(vector_length=>128),"
+                + " ) PRIMARY KEY (`id` ASC)"
+                ));
+    assertNotNull(ddl.hashCode());
```

Comment:
Not necessary to address this because the other tests have it, but it doesn't look like this assertion does anything because an `int` will never be `null`.